corporate/lib/stripe.py

```diff
@@ -988,6 +990,39 @@ def void_all_open_invoices(realm: Realm) -> int:
     return voided_invoices_count
 
 
+def downgrade_small_realms_behind_on_payments_as_needed() -> None:
+    customers = Customer.objects.all()
+    for customer in customers:
+        realm = customer.realm
+        if get_latest_seat_count(realm) >= 5:
+            continue
+
+        if get_current_plan_by_customer(customer) is None:
+            continue
+
+        due_invoice_count = 0
+        for invoice in stripe.Invoice.list(customer=customer.stripe_customer_id, limit=2):
+            if invoice.status == "open":
+                due_invoice_count += 1
+        if due_invoice_count < 2:
+            continue
+
+        void_all_open_invoices(realm)
```

Comment:
Let's add a comment here just noting that we've now decided to downgrade this custom and void all invoices, and the below will execute that action.

Reply:
Done.